plugin/pkg/admission/priority/admission.go

```diff
@@ -78,20 +79,20 @@ func (p *priorityPlugin) ValidateInitialization() error {
 }
 
 // SetInternalKubeClientSet implements the WantsInternalKubeClientSet interface.
-func (p *priorityPlugin) SetInternalKubeClientSet(client internalclientset.Interface) {
+func (p *priorityPlugin) SetExternalKubeClientSet(client kubernetes.Interface) {
 	p.client = client
 }
 
 // SetInternalKubeInformerFactory implements the WantsInternalKubeInformerFactory interface.
-func (p *priorityPlugin) SetInternalKubeInformerFactory(f informers.SharedInformerFactory) {
-	priorityInformer := f.Scheduling().InternalVersion().PriorityClasses()
+func (p *priorityPlugin) SetExternalKubeInformerFactory(f informers.SharedInformerFactory) {
+	priorityInformer := f.Scheduling().V1beta1().PriorityClasses()
 	p.lister = priorityInformer.Lister()
 	p.SetReadyFunc(priorityInformer.Informer().HasSynced)
 }
 
 var (
 	podResource           = api.Resource("pods")
-	priorityClassResource = scheduling.Resource("priorityclasses")
+	priorityClassResource = schedulingv1beta1.Resource("priorityclasses")
```

Comment:
where is this used?

Reply:
L131. this change shouldn't happen is all the obj contained in the admission attributes should be internal